cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/CassandraScheduler.java

```diff
@@ -88,6 +93,9 @@ public void resourceOffers(final SchedulerDriver driver, final List<Offer> offer
         if (LOGGER.isDebugEnabled()) {
             LOGGER.debug("> resourceOffers(driver : {}, offers : {})", driver, protoToString(offers));
         }
+
+        configuration.getSeedManager().syncSeeds(clock.now().getMillis());
```

Comment:
Correct, we don't want to the offer cycle to control or be impacted by the need of the seed manager to refresh it's info.